nautobot/core/runner/runner.py

```diff
@@ -234,14 +234,10 @@ def run_app(**kwargs):
 
         config_path = os.path.expanduser(args.config_path)
 
-        # Check if the config already exists; confirm w/ user
+        # Check if the config already exists; alert user and exit if exists.
         if os.path.exists(config_path):
-            resp = None
-            while resp not in ("y", "n", ""):
-                resp = input("File already exists at %r, overwrite? [yN] " % config_path).lower()
-                if resp != "y":
-                    print("Aborted!")
-                    return
+            print(f"A configuration already exists at {config_path}. Please backup and remove or choose another path.")
+            return
```

Comment:
I see that within this single function we are in some cases using `raise`, others using `parser.exit(1)` and in others simply `return`. It's not clear to me whether these variations are intentional and which would be most appropriate in this particular case.

@jathanism can you provide some input as someone who's more familiar with the history of this code?

Reply:
I don't understand how this is better? This doesn't give the option to overwrite the file and the existing implementation provides a user prompt. I do not personally find not having a prompt to be a desirable user experience.

Reply:
If we split the difference, I'd prefer to leave the prompt, but improve the warning message to make it more obvious before confirming that this will nuke your existing config e.g. "A configuration already exists at {config_path}. Please backup and remove or choose another path before proceeding!"

Reply:
It's making it harder for the user to shoot themselves in the foot a la #445. Can you think of a valid use case where someone really needs to overwrite an existing config with a new freshly generated one, without even preserving `SECRET_KEY` or similar parameters?

Reply:
Yeah okay. That's fair.

Reply:
Whoops, I never answered your question. 

@glennmatthews The original source of the runner code is http://github.com/dcramer/logan . In this context `return` is fine, since that's what it was already doing upon "abort".